bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -8,13 +9,31 @@ const getBreakouts = () => Breakouts.find({ parentMeetingId: Auth.meetingID })
   .fetch()
   .sort((a, b) => a.sequence - b.sequence);
 
-const getUsersNotAssigned = (users) => {
-  const breakouts = getBreakouts();
-  const breakoutUsers = breakouts
-    .reduce((acc, value) => [...acc, ...value.users], [])
-    .map(u => u.userId);
-  return users.filter(u => !breakoutUsers.includes(u.intId));
-};
+const getBreakoutUser = user => Users.find({
+  extId: new RegExp(`^${user.userId}`),
+  connectionStatus: 'online',
+}).fetch();
+
+const currentBreakoutUsers = userArray => userArray.length === 1;
+
+const filterBreakoutUsers = filter => users => users.filter(filter);
+
+const filterUsersNotAssigned = filterBreakoutUsers(currentBreakoutUsers);
+
+const mapUsersToNotAssined = mapFunction => users => users.map(mapFunction);
+
+const flatUsersArray = usersArray => usersArray.flat();
```

Comment:
@capilkey reported that `.flat()` is breaking the client on Edge`
@Tainan404 could you please use an alternative in a new PR?

Reply:
There's two options. The first is to include a polyfill from a package or just the function. The second would be to use an alternative function since the usage is pretty simple here. The MDN has some alternatives using concat() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat. I like think option of using a different function from flat() because adding a new node module just for this seems like overkill.